### PR TITLE
Re-former app

### PR DESCRIPTION
**Pull request by vichuge**
These projects gave us a chance to actually build some forms, both using nearly-pure HTML and then using the form helper methods that Rails provide.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(params[:user])
```
Comment (MarilenaRoque) [inline app/controllers/users_controller.rb]: - Please remove the commented line

File: app/views/users/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<h1>Edit User:</h1>
+<!--<form action="/users" method="post" accept-charset="UTF-8">
```
Comment (MarilenaRoque) [inline app/views/users/edit.html.erb]: - Use 'users_path' helper method

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,6 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :users
```
Comment (MarilenaRoque) [inline config/routes.rb]: - Use **only** to remove unnecessary routes

**Review by MarilenaRoque — CHANGES_REQUESTED** [review]
# Hello Team!!! 🚀 

### Excellent job, so far!

Just a few adjustments are necessary to achieve all the requirements for this project.

## Changes Required ♻️ 

- [x] Add info to your README.md about how to run the server and open your project in the browser.
- [x] Remove unnecessary commented code from your user's controller.
- [x] Please, use the keyword **only**, to create only the necessary routes for this project: new, create, edit and update.
- [x] Please replace the regular path for users_path helper method on the form tag.

Check my comments below for more specifications and feel free to contact me on Slack for any further information.

**Happy Coding!!!** :computer:

**_Marilena Roque
Full-Stack Developer
Code Reviewer at Microverse Inc.
[Portfolio](https://mariroqueproject.com/) | [Linkedin](https://www.linkedin.com/in/roquemarilena/) 
marilenas@id.uf.br_**

**Review by LamiaSristy — APPROVED** [review]
# Status: Approved. 💯 🥇 🎉 🎉

Hello Team @anapdh and @vichuge !!!

Your project is now approved. Congratulations on your hard work. 🛠️ 


You can merge the branch now, good luck with your next Project 🚀.

Feel free to reach me at @lamiasristy 🙋

**Pull request merged**